composer.lock

```diff
@@ -81,38 +81,40 @@
             "license": [
                 "MIT"
             ],
-            "time": "2019-11-27T14:58:02+00:00"
+            "time": "2019-12-10T11:03:53+00:00"
         }
     ],
     "packages-dev": [
         {
             "name": "doctrine/instantiator",
-            "version": "1.0.5",
+            "version": "1.3.0",
             "source": {
                 "type": "git",
                 "url": "https://github.com/doctrine/instantiator.git",
-                "reference": "8e884e78f9f0eb1329e445619e04456e64d8051d"
+                "reference": "ae466f726242e637cebdd526a7d991b9433bacf1"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/doctrine/instantiator/zipball/8e884e78f9f0eb1329e445619e04456e64d8051d",
-                "reference": "8e884e78f9f0eb1329e445619e04456e64d8051d",
+                "url": "https://api.github.com/repos/doctrine/instantiator/zipball/ae466f726242e637cebdd526a7d991b9433bacf1",
+                "reference": "ae466f726242e637cebdd526a7d991b9433bacf1",
                 "shasum": ""
             },
             "require": {
-                "php": ">=5.3,<8.0-DEV"
+                "php": "^7.1"
```

Comment:
Have the PHP version requirements changed? 🤔

Reply:
Not sure why that happened...

Reply:
Might be a random `composer update` / `composer install` from a PHP 7.* 🤔

Reply:
Yep, looks like it. It's just the dev requirements, the main requirements still allow both PHP versions.

Reply:
That's perfect then - all good for me